src/main/java/frc/robot/Constants.java

```diff
@@ -1,10 +1,32 @@
 package frc.robot;
 
-
 public final class Constants {
+    public static final double LOOP_PERIOD = 0.02; // loop period. [s]
+    public static final double g = 9.80665; // Gravity acceleration constant. [m/s^2]
+    public static final double UPPER_TARGET_HEIGHT = 2.64; // Height of upper target. [m]
+    public static final double PEBZNER_HEIGHT = 4.8; // Height of pebzner auditorium. [m]
```

Comment:
Why is this constant necessary?

Reply:
It's useful for calculations. We might want to calculate the trajectory of the ball.